### DISCUSSION TITLE
How to move project panel to right side

### DISCUSSION THREAD

**Discussion by nikivdev** [zed-industries/zed#6631] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

In VSCode i had my file explorer on right side:

<img width="1512" alt="image" src="https://github.com/zed-industries/community/assets/6391776/d171226d-ed07-4569-b459-3563f7381def">

I need to have same in Zed. How can I do it?

**Comment by nikivdev** [comment]
In https://zed.dev/blog/new-panel-system

I see 

<img width="828" alt="image" src="https://github.com/zed-industries/community/assets/6391776/9c1b9f41-5e84-47c1-beb0-ea8fbcd9f776">

So it should be possible but I don't get what config it is.

**Comment by nikivdev** [comment]
Ok I found it, right click on icon that opens project panel.

**Reply by SKRyanrr** [reply]
thanks

**Comment by tannal** [comment]
I changed the dock to the right but then ctrl B stop working.

**Reply by tannal** [reply]
> I changed the dock to the right but then ctrl B stop working.

Nevermind, it's ctrl alt B now.

**Reply by nilotpaul** [reply]
you can change the keymaps in keymap.json
```json
{
    "context": "Workspace",

    "bindings": {
      "ctrl-b": "workspace::ToggleRightDock"
    }
}
```

**Reply by tannal** [reply]
Good idea, thanks.

**Reply by PaulRBerg** [reply]
This should be done by default when moving the dock to the right. It's a bug otherwise

**Reply by olejorgenb** [reply]
> This should be done by default when moving the dock to the right. It's a bug otherwise

`ctrl-b` is not actually bound to _toggle project panel_ by default though? In the default bindings it's bound to `workspace::ToggleLeftDock` which is not advertised as toggling the project panel. (the "pretty" name is also "workspace: toggle left dock")

I don't think there's an dedicated action to toggle the project panel. Perhaps it should be. (you have a "project panel: toggle" but it doesn't hide the panel when toggled away (naturally))
